src/software/simulated_tests/kicker_double_touch.py

```diff
@@ -0,0 +1,66 @@
+import software.python_bindings as tbots_cpp
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class KickerDoubleTouch(Validation):
+
+    """Checks if the robot kicking the ball touches the ball again before another robot touches it."""
+
+    def __init__(self):
+        self.kicker_robot = None
+        self.kick_position = None
+        self.other_robot = None
+
+    def get_validation_status(self, world) -> ValidationStatus:
```

Comment:
based on the robocup rules (https://robocup-ssl.github.io/ssl-rules/sslrules.html#_double_touch), double touch is defined by the ball moving more than 0.05 m from its initial position and the kicker robot is still making contact with the ball unless it makes contact with another robot